db/drivers/mysql/grass-mesql.html

```diff
@@ -71,20 +71,17 @@ <h2>SEE ALSO</h2>
 </em>
 
 <h2>Credits</h2>
```

Comment:
Should this probably go into the `Author` section?

Reply:
Agreed: changed in 13a1ccce34005f15bec4dc8f64addb70441db389